packages/visualizer/src/extension/bridge.tsx

```diff
@@ -0,0 +1,205 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import { ChromeExtensionPageBrowserSide } from '@midscene/web/bridge-mode-browser';
+import { Button, Spin } from 'antd';
+import { useEffect, useRef, useState } from 'react';
+import './bridge.less';
+import { iconForStatus } from '@/component/misc';
+import dayjs from 'dayjs';
+
+interface BridgeLogItem {
+  time: string;
+  content: string;
+}
+
+const connectTimeout = 30 * 1000;
+const connectRetryInterval = 300;
+export default function Bridge() {
+  const activeBridgePageRef = useRef<ChromeExtensionPageBrowserSide | null>(
+    null,
+  );
+
+  const [bridgeStatus, setBridgeStatus] = useState<
+    'closed' | 'open-for-connection' | 'connected'
```

Comment:
The statuses here can be maintained within an enum in midscene/web/bridge-mode, enabling us to perceive the impact on both sides in the future.